tests/acceptance/features/webUIAccount/logout.feature

```diff
@@ -1,13 +0,0 @@
-@skip @ocis-konnectd-issue-14
-Feature: Logout users
-  As a user
-  I want to be able to logout of my account
-  So that I can protect my work and identity and be assured of privacy
-
-
-  Scenario:logging out
-    Given user "Alice" has been created with default attributes and without skeleton files
-    And user "Alice" has logged in using the webUI
-    When the user browses to the account page
-    And the user logs out of the webUI
-    Then the authentication page should be visible
```

Comment:
deleted this file because there are logout scenarios in other features
https://github.com/owncloud/web/blob/6df77c6b1e89f80e2ac6218827b181ad85c8e1ea/tests/acceptance/features/webUILogin/oauthLogin.feature#L23-L27
https://github.com/owncloud/web/blob/6df77c6b1e89f80e2ac6218827b181ad85c8e1ea/tests/acceptance/features/webUILogin/openidLogin.feature#L24-L28